build-wallet-electron.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+npm i && npm run build; 
+cd wallet-electron-ts; 
+npm i && npm run build; 
+npm run start; 
```

Comment:
Since this is just build file, i don't think we want to start the proxy. 

you could echo a helpful message at the end that says 

```
echo "Want to start the server? Run cd wallet-electron-ts && npm run start"
```

Reply:
Agree